src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeAccessServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.services;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.BDDMockito;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.EnvelopeAccessProperties;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.EnvelopeAccessProperties.Mapping;
+import uk.gov.hmcts.reform.bulkscanprocessor.exceptions.ForbiddenException;
+import uk.gov.hmcts.reform.bulkscanprocessor.exceptions.ServiceConfigNotFoundException;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+@RunWith(MockitoJUnitRunner.class)
+public class EnvelopeAccessServiceTest {
```

Comment:
Should we add a test case which verifies that no exception is thrown when service is allowed to update envelopes ?